github/repos_pages.go

```diff
@@ -93,6 +93,25 @@ func (s *RepositoriesService) GetLatestPagesBuild(owner string, repo string) (*P
 	return build, resp, err
 }
 
+// GetPageBuild fetches the specific build information for a GitHub pages site.
+//
+// GitHub API docs: https://developer.github.com/v3/repos/pages/#list-a-specific-pages-build
+func (s *RepositoriesService) GetPageBuild(owner string, repo string, ID int) (*PagesBuild, *Response, error) {
+	u := fmt.Sprintf("repos/%v/%v/pages/builds/%v", owner, repo, ID)
+	req, err := s.client.NewRequest("GET", u, nil)
+	if err != nil {
+		return nil, nil, err
+	}
+
```

Comment:
Have you tested this function against the GitHub API?

I ask because it is not clear to me from the documentation here:
https://developer.github.com/v3/repos/pages/#list-a-specific-pages-build
if the preview header `application/vnd.github.mister-fantastic-preview+json` is currently required for this API call to work.

Can you test this and see?
If it is required, can you please add that here with a TODO to remove it when the API is updated, along with a test for the preview header in the associated tests?